apps/req.c

```diff
@@ -844,21 +841,16 @@ int req_main(int argc, char **argv)
                                "Warning: Signature key and public key of cert do not match\n");
                 ERR_pop_to_mark();
             }
-            X509V3_set_nconf(&ext_ctx, req_conf);
 
             /* Add extensions */
             if (extsect != NULL
-                && !X509V3_EXT_add_nconf(req_conf, &ext_ctx, extsect, new_x509)) {
-                BIO_printf(bio_err, "Error adding x509 extensions from section %s\n",
-                           extsect);
+                && !do_EXT_add_nconf(req_conf, &ext_ctx, extsect, new_x509,
+                                     "Error adding x509 extensions from section %s\n"))
                 goto end;
-            }
             if (addext_conf != NULL
-                && !X509V3_EXT_add_nconf(addext_conf, &ext_ctx, "default",
-                                         new_x509)) {
-                BIO_printf(bio_err, "Error adding x509 extensions defined via -addext\n");
+                && !do_EXT_add_nconf(addext_conf, &ext_ctx, "default", new_x509,
+                                     "Error adding x590 extensions defined via -addext%0s\n"))
```

Comment:
You have %0s here as well.

Reply:
Oops, fixed, as well as a typo in this message string (x590 vs. x509).